docs/articles/nunit/writing-tests/assertions/assertion-models/classic.md

```diff
@@ -59,3 +55,7 @@ Additional assertions are provided by the following classes:
 ## See Also
 
 * [Constraint Model](xref:constraintmodel)
+
+## Notes
+
+* The exception-family of classic asserts have not yet been moved to the legacy namespace.  They will be moved and renamed in a later release. They have equivalents in the constraint model.
```

Comment:
As mentioned [here](https://github.com/nunit/nunit/issues/4416#issuecomment-1636774876) there is no general constraint that works (nicely) on the _exception_ instance:
```csharp
Assert.That(ThrowException, Throws.InstanceOf<InvalidOperationException>().With
                                  .Matches<InvalidOperationException>(x => x.StackTrace is not null && x.StackTrace.Contains("ThrowException")));
```
It doesn't read nice (With.Matches) and requires the duplication of the type name. The latter is due to the more fundamental non-generic nature of the constraints. Most of us tried to change this and have failed. Maybe on a small specific constraint like this it could actually work.

Reply:
Ok, so then we'll just keep them "as-is" ?

Reply:
I suggest for now to say that `They have not yet full equivalents in the constraint model.`